packages/valory/skills/abstract_round_abci/tests/test_utils.py

```diff
@@ -158,21 +148,14 @@ def test_to_int_positive() -> None:
     assert to_int(542, 2) == 54200
 
 
-@pytest.mark.skip
-def test_fuzz_to_int() -> None:
+@given(
+    st.floats(max_value=1e16, min_value=-1e16, allow_nan=False, allow_infinity=False),
+    st.integers(min_value=0, max_value=20),
+)
+def test_fuzz_to_int(estimate: float, decimals: int) -> None:
```

Comment:
this is not a proper range to test, I made a mistake here, addressing

Reply:
okay this test will fail even when properly done, because our `to_int` method does not deal with `e` notation